decimal.go

```diff
@@ -52,6 +52,14 @@ var DivisionPrecision = 16
 // silently lose precision.
 var MarshalJSONWithoutQuotes = false
 
+// StringTrimTrailingZeros should be set to false if you want the decimal stringify without zeros trailing.
+// By default, when decimal is output as a string (for example, in JSON), zeros are truncated from it (2.00 -> 2, 3.11 -> 3.11, 13.000 -> 13).
+// But this logic can be changed by this variable.
+// For example, if you have numeric(10,2) values stored in your database,
+// and you want your API response to always be given 2 decimal places (even 2.00, 3.00, 17.00 [not 2,3,17]),
+// then this variable is a great way out.
+var StringTrimTrailingZeros = true
```

Comment:
I also wonder whether we should introduce another config var for specifying NewFromString behaviour. The current implementation does not trim zeroes during the initialization from string, but we saw in the past that some developers would like to see such an option.
Unfortunately, we cannot use this newly introduced variable for both of these cases as we would break backward compatibility.